app/views/mailers/verification_mailer/account_ownership_verification_email.html.erb

```diff
@@ -0,0 +1,18 @@
+<% host = "#{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}" %>
+<% verify_url = "#{host}/verify_email_ownership?confirmation_token=#{@user.email_authorizations.order('created_at DESC').first.confirmation_token}&username=#{@user.username}" %>
```

Comment:
IMO we should get the token from the controller and put it in an instance variable to render here.